spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -208,6 +208,12 @@ def types
       end
     end
 
+    context 'sensitive' do
+      it 'translates to PSensitiveType' do
+        expect(calculator.infer(PSensitiveType::Sensitive.new("hunter2")).class).to eq(PSensitiveType)
+      end
+    end
```

Comment:
I'm also considering treating this as a limitation - if you want to use sensitive information, your ability to deeply validate data goes away. This is less of an issue with the Sensitive type since we're still handling sensitive information, but when we start ingesting Encrypted information we absolutely cannot expose information about the encrypted information. Exposing the type information alone is somewhat risky; including any information in the associated type information is actively dangerous.